README.md

```diff
@@ -15,7 +15,7 @@ wgpu-rs can target both the natively supported backends and WASM directly.
 ## Gallery
 
 ![Cube](etc/example-cube.png) ![Shadow](etc/example-shadow.png) ![MipMap](etc/example-mipmap.png) ![Skybox](etc/example-skybox.gif)
-![vange-rs](etc/vange-rs.png) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
+![vange-rs](etc/vange-rs.png) ![Blub](etc/blub-fluid-small.gif) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
```

Comment:
I'm concerned about this being in each and every checkout of the code.
Let's instead try to cut this gallery a bit (only non-animated and small images), and expand the one on `wgpu.rs`.
How do you feel about targeting the "gh-pages" branch for this?